TRANSLATIONS.md

```diff
@@ -12,6 +12,14 @@ Translated lessons should follow this naming convention:
 
 where _[language]_ is a two letter language abbreviation following the ISO 639-1 standard (e.g. `README.es.md` for Spanish and `README.nl.md` for Dutch).
 
+When translatinng lessons, link the other translations to your translated lesson by using Shields as shown 
```

Comment:
This should probably say to add the just to the initial readme, there's no need to have it on every lesson, just the lop level readme to help navigate into the first translated file